src/buildercore/terraform.py

```diff
@@ -123,18 +125,22 @@ def render_fastly(context):
             backends.append(_fastly_backend(
                 backend['hostname'],
                 name=name,
-                request_condition=backend_condition_name
+                request_condition=backend_condition_name,
+                shield=backend['shield'].get('pop')
             ))
```

Comment:
this branch, which is for multiple backends, doesn't seem to add to `all_allowed_subdomains`?

Reply:
Read a comment about shielding being incompatible with multiple backends, it's probably not the case anymore if `shield` is added here?

Reply:
Originally was going to make the default `shield` and `backends` incompatible (with the former referring solely to the automatic backend), but now you can disable/overwrite in each backend as needed.